docs/synapse/userguides/storm_adv_libs.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true,
+    "hideOutput": true
+   },
+   "outputs": [],
+   "source": [
+    "import os, sys\n",
+    "try:\n",
+    "    from synapse.lib.jupyter import *\n",
+    "except ImportError as e:\n",
+    "    # Insert the root path of the repository to sys.path.\n",
+    "    # This assumes the notebook is located three directories away\n",
+    "    # From the root synapse directory. It may need to be varied\n",
+    "    synroot = os.path.abspath('../../../')\n",
+    "    sys.path.insert(0, synroot)\n",
+    "    from synapse.lib.jupyter import *"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true,
+    "hideOutput": true
+   },
+   "outputs": [],
+   "source": [
+    "# Create a cortex\n",
+    "core = await getTempCoreCmdr()"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. highlight:: none\n",
+    "\n",
+    ".. _storm-adv-libs:\n",
+    "\n",
+    "Storm Reference - Advanced - Libraries\n",
+    "======================================\n",
+    "\n",
+    "A **partial** list of libraries available in Synapse is provided below. These libraries represent a subset most likely to be referenced by users.\n",
+    "\n",
+    ".. _storm-lib:\n",
+    "\n",
+    "$lib\n",
+    "----\n",
+    "\n",
+    ":ref:`vars-global-lib` is a global variable used to access Storm's libraries. ``$lib`` is not used by itself, but with the library functions below to perform various common operations in Storm."
```

Comment:
We should probably break this into three headings:
`$lib.<func>` where `func()` does a useful thing.
`$lib.<cgor>` where `ctor()` makes a new Storm Primitive object. `set`, `list`, `text`, and `dict`.
$lib.<library name>` where `library name` is the library which contains a bunch of methods.